pkgs/development/tools/github-changelog-generator/Gemfile

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+source "https://rubygems.org"
+
+gem "github_changelog_generator", git: "https://github.com/github-changelog-generator/github-changelog-generator/", ref: "v1.14.3"
```

Comment:
Why not just use the version from RubyGems?

Reply:
will fix that